.gitignore

```diff
@@ -10,4 +10,6 @@ dnf-json
 /gen-oscap
 local.env
 __debug*
-coverage.txt
+coverage*
+
```

Comment:
(super nitick) extra newlines here that we probably don't need (but they also don't hurt so fine to ignore)